.nvmrc

```diff
@@ -0,0 +1 @@
+10
```

Comment:
not 14?

Reply:
No, annoyingly - it's 10.

Reply:
Never been updated and don't want to deal with that upgrade in this PR - better to at least state what it was built against